scripts/apps/authoring/versioning/history/views/history.html

```diff
@@ -2,7 +2,7 @@
     <ul class="history-list">
         <li class="item"
             ng-class="{selected: selected === version}"
-            ng-repeat="version in versions track by version._current_version"
+            ng-repeat="version in versions"
```

Comment:
If I am not wrong the _current_version should be unique for versions of the same item. So if track by fails here, it maybe because of some other problem.